Frameworks/include/StringHelpers.h

```diff
@@ -465,9 +466,22 @@ inline TSource ContainerFromDelimitedString(_In_ const std::wstring& delimitedSt
     return result;
 }
 
-// Returns a a vector from a delmited string
+// Returns a vector from a delmited string
 inline std::vector<std::wstring> VectorFromDelimitedString(_In_ const std::wstring& delimitedString, wchar_t delimiter) {
     return ContainerFromDelimitedString<std::vector<std::wstring>>(delimitedString, delimiter);
 }
 
+#ifdef __OBJC__
```

Comment:
it's actually needed. do you want to grab the build error for you?

---
In reply to: [90742443](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90742443) [](ancestors = 90742443)